pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -34,6 +34,9 @@ type AnacondaInstallerISOTree struct {
 	Keyboard *string
 	Timezone *string
 
+	// setup network automatically in target
+	NetworkOnBoot bool
```

Comment:
@achilleas-k I guess this should also be KickstartNetworkOnBoot now just like in https://github.com/osbuild/images/pull/614 ?